packages/backend/src/payment-method/ilp/connector/core/telemetry.ts

```diff
@@ -0,0 +1,61 @@
+import { TelemetryService } from '../../../../telemetry/service'
+import { IlpResponse } from './middleware/ilp-packet'
+import { ValueType } from '@opentelemetry/api'
+
+export function incrementPreparePacketCount(
+  unfulfillable: boolean,
+  prepareAmount: string,
```

Comment:
@koekiebox  The prepare packet has the amount set as a string, so I was just staying true to that. It gives the function a simple signature then since we don't need to check if Number(prepare.amount) before calling this function since it was explicitly designed to abstract all this reasoning into one simple call.

Reply:
@mkurapov, do you have any feelings about this and which way you'd like me to have this implemented since Jason is on vacation this week.